rsmanage/setup.py

```diff
@@ -3,10 +3,10 @@
 setup(
     name="rsmanage",
     version="0.1",
-    py_modules=["rsmanager"],
+    py_modules=["rsmanage"],
```

Comment:
Wait, what?  Why change this name??

Reply:
This means we can use `python -m rsmanage` to run this module (as opposed to `python -m rsmanager`). The name change makes it match the command-line entry point `rsmanage`.

Since this is run from the server in in the `default/delete` endpoint, we need to execute this as `settings.python_interpreter -m rsmanage` and keeping the name consistent seemed best to me.